plugins/chat/assets/javascripts/discourse/lib/chat-preload-link.js

```diff
@@ -0,0 +1,16 @@
+export function addPreloadLink(url) {
```

Comment:
I think this function should probably accept a list of URLs instead to be more efficient. We can then make use of [element.append()](https://developer.mozilla.org/en-US/docs/Web/API/Element/append) to append all the new nodes in one go such that the browser only has to render once and not once per URL.

Reply:
Will see how it works in practice and we can iterate yes

Reply:
There's other thing I might change in the API, like accepting options, eg: fetchPriority